python/cac_tripplanner/destinations/models.py

```diff
@@ -48,6 +57,32 @@ class Meta:
     def __unicode__(self):
         return self.name
 
+class Event(models.Model):
+    """Represents an event, which has a start and end date"""
+
+    class Meta:
+        ordering = ['priority', '-start_date']
+
+    name = models.CharField(max_length=50)
+    website_url = models.URLField(blank=True, null=True)
+    description = RichTextField()
+    start_date = models.DateTimeField()
+    end_date = models.DateTimeField()
```

Comment:
Was thinking we would probably ignore the time portion of the date range on the front end, but it would be valuable to store in the case of a single-day event. Hours of operation can simply go in the description field, since we're not going to try to filter on that (or at least not yet).

Reply:
Yeah, makes sense.